doc/TockBinaryFormat.md

```diff
@@ -270,8 +272,10 @@ The `Main` element has three 32-bit fields:
   * `init_offset` the offset in bytes from the beginning of binary payload
     (i.e. the actual application binary) that contains the first instruction to
     execute (typically the `_start` symbol).
-  * `protected_size` the amount of flash, in bytes, after the header, to
-    prevent the process from writing to.
+  * `protected_size` the size of the protected region in bytes. The protected
+    region begins at the start of the app image and ends just before the
+    compiled app binary. Processes may not write to the protected region. The
+    TBF headers should be contained within the protected region.
```

Comment:
At the risk of being too pedantic, are _may_ and _should_ really appropriate here? This might indicate that this could be allowed on some platforms and given our strict requirements around memory protection for platforms we target, I can't come up with good usecases to violate these invariants. Perhaps _must_ and _must_ are more appropriate here?

Reply:
Since this isn't a TRD, those words don't have precise meanings, unfortunately. However, I agree there is some ambiguity. How about

"Processes do not have write access to the protected region. TBF headers are contained in the protected region."

Reply:
Reading it again, my original wording sounds like it is asking processes to not write the protected region, when I intended to say they would be prevented from writing the protected region. I think Phil's wording is much clearer, so I switched to his wording.

Reply:
Right, I think @phil-levis' version is great. Thanks for changing it!

Reply:
Right, I think @phil-levis' version is great. Thanks for changing it!